tests/console/journalctl.pm

```diff
@@ -38,6 +38,11 @@ use constant {
     SEALING_DELAY      => 10
 };
 
+# If the daemon is stopped uncleanly, or if the files are found to be corrupted, they are renamed using the ".journal~" suffix
+sub corrupted_logfiles {
+    return (script_output('find /var/log/journal/$(cat /etc/machine-id) -iname "*.journal~" -type f -print0') ne "");
```

Comment:
maybe you want to add `2>/dev/null` e.g. if that directory doesn't exists.

I don't think you need the `()` around this statement.

Reply:
```
    if (is_opensuse && !is_leap('>=15.3')) {
        assert_script_run 'rpm -q systemd-logger';
        assert_script_run "rpm -q --conflicts systemd-logger | tee -a /dev/$serialdev | grep syslog";
    } else {
        script_run("journalctl --boot=-1 | grep 'no persistent journal was found'") or die "Persistent journal should not be enabled by default on SLE!\n";
        assert_script_run "mkdir ${\ PERSISTENT_LOG_DIR }";
        assert_script_run "systemd-tmpfiles --create --prefix ${\ PERSISTENT_LOG_DIR }";
```
It should exist as it is either provided by the package `systemd-logger` in opensuse or it will be created by `assert_script_run "mkdir ${\ PERSISTENT_LOG_DIR }";` Frankly the `machine-id` part might be superfluous on the second thought. 

Regarding the `()` I am always thinking of `print ($foo & 255) + 1, "\n";` vs `print(($foo & 255) + 1, "\n");`. I know it is not entirely the same case, but feels safer to me. You made me confident to remove it from there! :)

Reply:
Yeah, I think that `perl` has the possibility to **not** use brackets for function calls make this language more readable :)